reana/config.py

```diff
@@ -266,6 +266,9 @@
 TIMEOUT = 900
 """Maximum timeout to wait for results when running demo analyses in CI."""
 
+KIND_IMAGE_VERSION = "v1.30.2"
+"""Kind Docker image version. Should be compatible with kubeVersion in HELM_VERSION_FILE."""
```

Comment:
nitpicking:
```suggestion
"""Kind node Docker image version. Should be compatible with kubeVersion in HELM_VERSION_FILE."""
```

PS: we will not need this default value if we decide to make the flag optional, see other comment